### PR TITLE
Array conversion functions

### PR DESCRIPTION
**Pull request by skuzmich**
- **Fix issues with :generator:generateWasmStdlibFromIDL task**
- **Add extension methods for primitive JS array conversion**

### CODE REVIEW

File: src/wasmJsTest/kotlin/arrayCopyTest.kt [hunk 1]
```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2010-2024 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+ */
+
+// NOTE: THIS FILE IS AUTO-GENERATED, DO NOT EDIT!
+// See github.com/kotlin/dukat for details
```
Comment (igoriakovlev) [inline src/wasmJsTest/kotlin/arrayCopyTest.kt:7]: This is not correct, right?

File: generator/src/main/kotlin/helpers/generate.kt [hunk 2]
```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
```
Comment (igoriakovlev) [inline generator/src/main/kotlin/helpers/generate.kt:2]: Should be 2024?

File: src/wasmJsMain/kotlin/arrayCopy.kt [hunk 3]
```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2010-2024 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+ */
+
+// NOTE: THIS FILE IS AUTO-GENERATED, DO NOT EDIT!
+// See github.com/kotlin/dukat for details
```
Comment (igoriakovlev) [inline src/wasmJsMain/kotlin/arrayCopy.kt:7]: Not correct comment?

**Review by igoriakovlev — CHANGES_REQUESTED** [review]

File: src/wasmJsTest/kotlin/arrayCopyTest.kt (hunk 1 above)
Comment (skuzmich) [reply src/wasmJsTest/kotlin/arrayCopyTest.kt]: Fixed

File: generator/src/main/kotlin/helpers/generate.kt (hunk 2 above)
Comment (skuzmich) [reply generator/src/main/kotlin/helpers/generate.kt]: Yes

File: src/wasmJsMain/kotlin/arrayCopy.kt (hunk 3 above)
Comment (skuzmich) [reply src/wasmJsMain/kotlin/arrayCopy.kt]: Yes, fixed

**Comment by igoriakovlev** [comment]
Please sqash

**Pull request merged**